lib/utils.js

```diff
@@ -69,6 +69,10 @@ const findLocationOf = (keys, ast, initialFormat) => {
     info = node.loc;
   }
 
+  if (!info || !info.start) {
+    throw new Error('Invalid `info` object.');
```

Comment:
please throw ParserError here. All the types are mentioned here https://github.com/asyncapi/parser-js/#error-types

although I'm not sure error is a good solution here. If info is missing we are just not able to return in the error information about the error location, but jsonPointer can be returned. And if you check `start` then why not `end`?

why did you choose throwing an Error?

Reply:
Firstly I want use ParserError and I didn't know which type choose.

You're right, return `jsonPointer` is a better idea than throw error, sorry for that. In this solution I only check start, because then in code `info.end` is checked in ternary operators.

I will adjust method to return jsonPointer.

Reply:
@magicmatatjahu lgtm, great!